Purchases/Networking/Operations/Handling/SubscriberAttributesMarshaller.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  SubscriberAttributesMarshaller.swift
+//
+//  Created by Joshua Liebowitz on 11/18/21.
+
+import Foundation
+
+class SubscriberAttributesMarshaller {
+
+    func subscriberAttributesToDict(subscriberAttributes: SubscriberAttributeDict) -> [String: Any] {
```

Comment:
thoughts on more specifically typing the return value? to [String: [String: Any]?

Reply:
You are right, we can do that! Thank you for pointing this out.